js/build/index.html

```diff
@@ -19,7 +19,11 @@
     Learn how to configure a non-root public URL by running `npm run build`.
   -->
   <title>React App</title>
-<link rel="shortcut icon" href="/favicon.ico"></head>
+  <link rel="stylesheet" href="https://amptesting.sites.stanford.edu/core/misc/dropbutton/dropbutton.css"/>
+  <link rel="stylesheet" href="https://amptesting.sites.stanford.edu/core/themes/stable/css/core/dropbutton/dropbutton.css?q6wkff"/>
+  <link rel="stylesheet" href="https://amptesting.sites.stanford.edu/core/themes/classy/css/components/dropbutton.css?q6wkff"/>
+  <link rel="stylesheet" href="https://amptesting.sites.stanford.edu/core/themes/seven/css/components/dropbutton.component.css"/>
+<link rel="shortcut icon" href="/favicon.ico"><link href="/../css/../../css/react_paragraphs.field_widget.css" rel="stylesheet"><link href="/../css/../../css/react_paragraphs.field_formatter.css" rel="stylesheet"></head>
```

Comment:
This is one ugly href

Reply:
I know right. I tried to make it better but since we dont use this file it's kinda a wasted effort